src/plugins/intel_cpu/src/config.cpp

```diff
@@ -92,10 +94,10 @@ void Config::readProperties(const std::map<std::string, std::string> &prop, cons
         } else if (hintsConfigKeys.end() != std::find(hintsConfigKeys.begin(), hintsConfigKeys.end(), key)) {
             perfHintsConfig.SetConfig(key, val);
         } else if (key == ov::hint::enable_cpu_pinning.name()) {
-            if (val == PluginConfigParams::YES) {
+            if (val == InferenceEngine::PluginConfigParams::YES) {
```

Comment:
`InferenceEngine::PluginConfigParams::YES` should be removed. Please use `.as<bool>()` instead!

Reply:
I'm not sure whether all legacy metric/config string value has been converted into ov::Any in Core level, so put cpu plugin properties  migration into a separated PR: https://github.com/openvinotoolkit/openvino/pull/20022